src/backend/base/langflow/langflow_launcher.py

```
@@ -0,0 +1,69 @@
+    env = os.environ.copy()
+    # Required for macOS to avoid fork safety issues.
+    # Error: """
+    # When fork() is called, NSCheapMutableString initialize may be in progress in another thread.
+    # This cannot be safely called or ignored in the fork() child process, causing a crash.
+    # To debug, set a breakpoint on objc_initializeAfterForkError
+    # """
+    env["OBJC_DISABLE_INITIALIZE_FORK_SAFETY"] = "YES"
+
+    # If no command specified, default to 'run'
+    args = sys.argv[1:] if len(sys.argv) > 1 else ["run"]
+    command = ["uv", "run", "python", "-m", "langflow.__main__"] + args
```

Ruff (0.11.9)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)